mitiq/cdr/cdr_execution.py

```diff
@@ -0,0 +1,202 @@
+                               but more successful the mitigation.
+        ansatz: the function to map noisy to exact data. Takes array of noisy
+                and data and parameters returning a float.
+        num_parameters: the number of paramters the ansatz takes.
+        scale_noise: optional argument containing a user defined function on
+                     how to increase the noise. If this argument is given then
+                     the mitigation method will be vnCDR.
+        scale_factors: factors by which to scale the noise, should not
+                               include 1 as this is just the original circuit.
+        kwargs: Available keyword arguments are:
+
+            TRAINING SET CONSTRUCTION OPTIONS:
+
+            - method_select (string): specifies the method used to select the
+                                      non-Clifford gates to replace when
+                                      constructing the near-Clifford training
+                                      circuits. Available options are:
+                                            ['uniform', 'gaussian']
+            - method_replace (string): specifies the method used to replace the
+                                      selected non-Clifford gates with a
+                                      Clifford when constructing the
+                                      near-Clifford training circuits.
+                                      Available options are:
+                                        ['uniform', 'gaussian', 'closest']
+            - sigma_select (float): Width of the Gaussian distribution used for
+                                    ``method_select='gaussian'``.
+            - sigma_replace (float): Width of the Gaussian distribution used
+                                     for ``method_replace='gaussian'``.
+            - random_state (int): seed for sampling.
+    Returns: The tuple (raw_expectations, mitigated_expectations)
+             corresponding to the many raw expectation values (at different
+             noise levels) and the associated mitigated values.
+
+    .. [Czarnik2020] : Piotr Czarnik, Andrew Arramsmith, Patrick Coles,
+        Lukasz Cincio, "Error mitigation with Clifford quantum circuit
+        data," (https://arxiv.org/abs/2005.10189).
+    .. [Lowe2020] : Angus Lowe, Max Hunter Gordon, Piotr Czarnik,
+        Andrew Arramsmith, Patrick Coles, Lukasz Cincio,
+        "Unified approach to data-driven error mitigation,"
+        (https://arxiv.org/abs/2011.01157)."""
+    # Extracting kwargs:
+    method_select = kwargs.get("method_select", "uniform")
+    method_replace = kwargs.get("method_replace", "closest")
+    random_state = kwargs.get("random_state", None)
+    training_set_generation_kwargs_keys = ["sigma_select", "sigma_replace"]
+    kwargs_for_training_set_generation = {
+        key: kwargs.get(key) for key in training_set_generation_kwargs_keys
+    }
```

Comment:
```suggestion
    kwargs_for_training_set_generation = {
        "sigma_select": kwargs.get("sigma_select"),
        "sigma_replace": kwargs.get("sigma_replace"),
    }
```
Unless I am missing something, I think this can be simplified.